pennylane_lightning/lightning_tensor/_measurements.py

```diff
@@ -373,15 +372,7 @@ def _measure_with_samples_diagonalizing_gates(
         wires = reduce(sum, (mp.wires for mp in mps))
 
         def _process_single_shot(samples):
-            processed = []
-            for mp in mps:
-                res = mp.process_samples(samples, wires)
-                if not isinstance(mp, CountsMP):
-                    res = qml.math.squeeze(res)
-
-                processed.append(res)
-
-            return tuple(processed)
+            return [mp.process_samples(samples, wires) for mp in mps]
```

Comment:
```suggestion
            return tuple(mp.process_samples(samples, wires) for mp in mps)
```

To continue returning a tuple as opposed to switching to a list